packages/bento-frontend/src/store/index.js

```diff
@@ -6,14 +6,17 @@ import layout from '../components/Layout/LayoutState';
 import dashboard from '../pages/dashboard/dashboardState';
 import stats from '../components/Stats/StatsState';
 import { sideBarReducerGenerator } from '../bento-core/FacetFilter/store/reducers/SideBarReducer';
+import { cartReducerGenerator } from '../bento-core/Cart/store/reducers';
 import { LocalFindReducerGenerator } from '@bento-core/local-find';
 
-const { statusReducer } = sideBarReducerGenerator();
 const { localFind } = LocalFindReducerGenerator();
+const { statusReducer } = sideBarReducerGenerator();
+const { cartReducer } = cartReducerGenerator();
```

Comment:
Aren’t we injecting cart reducers at run time?

Reply:
@injectReducer method is used.